src/types/ErrorType.ts

```diff
@@ -0,0 +1,5 @@
+export enum FilterType {
+  All = 'All',
+  active = 'Active',
+  completed = 'Completed',
+}
```

Comment:
Don't need to duplicate enum with the same values 
<img width="459" alt="image" src="https://github.com/user-attachments/assets/2c8602fc-4814-42c7-abe6-a852f746f932" />
```suggestion
```